content/articles/generic-programming-in-c#/index.md

```diff
@@ -0,0 +1,182 @@
+
+### Introduction
+
+In any modern system, data handling is one of the major and vital goals. The way data is handled is what makes the total difference. Some systems go for generic one which includes lists, dictionaries, and hash.
+
+Others prefer non-generic which are arrays, array lists, and bit arrays. Non- generic programming represents an ordered collection of an object that can be indexed individually. As a matter of fact, generic programming is strongly typed while non- generic programming is not strongly typed, which means generic programming enforces strict rules on blending of values with different data types.
+
+But due to the great advancement of technology in the 21st century, many software systems require to store large groups of entities
```

Comment:
> But due to the great advancement of technology in the 21st century, many software systems require to store large groups of entities
rather than individual entry. 

This sentence is a little long, consider shortening it or splitting it up. Also, you should pluralize *individual entry* (i.e. change it to *individual entries*).